src/main/java/com/leshchenko/springboot/service/ReviewParserFromCsvRecord.java

```diff
@@ -0,0 +1,11 @@
+package com.leshchenko.springboot.service;
+
+import com.leshchenko.springboot.model.Review;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVRecord;
+
+public interface ReviewParserFromCsvRecord {
```

Comment:
А ми можемо цей інтерфейс зробити більш загальним? Типу: 
```suggestion
public interface CsvRecordParser<T> {
```

Reply:
done